src/exceptions/KiCad-libraries-exception.xml

```diff
@@ -1,5 +1,5 @@
 <SPDXLicenseCollection xmlns="http://www.spdx.org/license">
-  <license isOsiApproved="false" licenseId="KiCad-libraries-exception" listVersionAdded="3.17" name="KiCad Libraries Exception">
+  <exception isOsiApproved="false" licenseId="KiCad-libraries-exception" listVersionAdded="3.17" name="KiCad Libraries Exception">
```

Comment:
Maybe also remove isOsiApproved and listVersionAdded?

Reply:
It was the `isOsiApproved` - looks like that attribute isn't allowed for exceptions.